src/components/Database/DataViewer/CloneDialog.tsx

```diff
@@ -83,7 +102,7 @@ export const CloneDialog = React.memo<Props>(function CloneDialog$({
     Object.entries(form).forEach(([key, value]) => {
       data[key] = JSON.parse(value);
     });
-    realtimeRef.parent!.child(newKey).set(data);
+    realtimeRef.root.child(newKey).set(data);
```

Comment:
Now that its a path and not a key, update the UI to say something like `Destination (path)`